apps/epl/src/epl.erl

```diff
@@ -23,24 +27,73 @@
 %% ===================================================================
 %% API functions
 %% ===================================================================
+
+%% @doc Lookups for given `Key' in epl_priv ets.
+-spec lookup(Key :: term()) -> [tuple()].
 lookup(Key) ->
     ets:lookup(epl_priv, Key).
 
+%% @doc Adds calling process to every epl_tracers' subscribers list.
```

Comment:
For each node in the cluster we spawn a gen_server process, which is implemented in the epl_tracer module. Process calling this function is added to the subscribers list, which is kept in the state of the ep_tracer gen_server.